### PR TITLE
Added new workbook for multi-instance assessment for Azure Arc for SQ…

### PR DESCRIPTION
**Pull request by nhebbar2011**
…L Servers

## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
Forking off the workbook used by the SQL VM team for assessment, since the Arc team supports best practices assessment for multiple instances and the workbook should be accordingly modified.
Made some spacing changes in the tags column in different tables. These spacing changes are applicable for the SQLVM workbook and the Arc workbook
There are no gallery changes
### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes

* [ ] ensure your template has a corresponding gallery entry in the gallery folder
NA
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

**Comment by nhebbar2011** [comment]
Spacing in Tags column
spacing_in_tags_column

Drop down for instance name
drop_down_for_instance_name

**Review by gardnerjr — APPROVED** [review]
only approving the codeowners parts, don't have resources to validate the other query changes, should be done by owning team

File: Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook [hunk 1]
```diff
@@ -554,7 +554,7 @@
...
... Sev = toint(asmt[8])\r\n| where AsmtId == \"{AsmtId}\" \r\nand (set_has_element(dynamic([{filterCheckId}]), CheckId) or \"{filterCheckId}\" == \"'*'\")\r\nand (set_has_element(dynamic([{filterTargetName}]), TargetName)  or \"{filterTargetName}\" == \"'*'\")\r\nand set_has_element(dynamic([{filterSev}]), Sev)\r\nand (array_length(set_intersect(TagsArr, dynamic([{filterTags}]))) > 0 or \"{filterTags}\" == \"'*'\")\r\n| extend Category = case(array_length(set_intersect(TagsArr, dynamic([{typeSystem:value}]))) > 0, '0',\r\narray_length(set_intersect(TagsArr, dynamic([{typeAdmins:value}]))) > 0, '1',\r\narray_length(set_intersect(TagsArr, dynamic([{typeDevs:value}]))) > 0, '2',\r\n'3')\r\n| where (Sev >= 0 and array_length(selectedTotSev) == 0 or Sev in (selectedTotSev))\r\nand (Category in (selectedCategories) or array_length(selectedCategories) == 0)\r\n| project TargetType, TargetName, Severity, Tags=strcat_array(array_slice(TagsArr, 1, -1), ', '), CheckId, SeverityCode = toint(Sev), Message\r\n| distinct * \r\n| summarize Cnt = count() by SeverityCode, Severity, Tags, CheckId\r\n| project Severity, Tags, CheckId, Cnt, SeverityCode\r\n| order by SeverityCode desc, Cnt desc\r\n\r\n",
```
Comment (shivgupta1997) [inline Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook:557]: Why are modifying SQL VM notebook?

**Review by shivgupta1997 — APPROVED** [review]

File: Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook (hunk 1 above)
Comment (nhebbar2011) [reply Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook:557]: There was some spacing issue bug that was reported. ​ The list of tags was not getting separated by spaces.
The change is visible in the screenshot.

**Review by naxing123 — APPROVED** [review]

**Pull request merged**